If reauth is required, show the full command-line path of the binary

rogerhu opened the pull request:
The error message that gets printed out tells me to login again. However, it doesn't show the full path of the binary that was used to run the credential helper.

minor-fixes reviewed (commented):
Instructions say to put `engflow_auth` on `$PATH`, so IMO it's fair and more user-friendly to carry this assumption.

It's a bit less convenient for us, if we're working with dev builds - we have to know what we're doing there, instead of following the error message blindly. But I'd still prefer the simpler perceived UX for users.

---

On second thought - maybe we can see if the current application is on `$PATH`, and display the full path if it isn't?

jayconrod commented:
I think what @rogerhu is doing here is the best option: print `os.Args[0]`.

If the user installed in `PATH` and invoked as `engflow_auth`, great, we report that.

If the user invoked with an absolute path (whether or not `engflow_auth` is in `PATH`), then we report that full path.

rogerhu commented:
> On second thought - maybe we can see if the current application is on `$PATH`, and display the full path if it isn't?

Addressing your concerns. Pushed some changes. =)

rogerhu commented:
> I think what @rogerhu is doing here is the best option: print `os.Args[0]`.
> 
> If the user installed in `PATH` and invoked as `engflow_auth`, great, we report that.
> 
> If the user invoked with an absolute path (whether or not `engflow_auth` is in `PATH`), then we report that full path.

Ok, I'll keep it as is. =)
